life_hc/MySQL/Items/fn_spikeStrip.sqf

```diff
@@ -30,7 +30,7 @@ if (hc_spikes isEqualTo []) then { //start monitoring spikestrips
                 deleteVehicle _x;
                 _toDelete pushBack _forEachIndex;
             } else {
-                if (isNull _object) then {
+                if (isNull _x) then {
                     deleteVehicle _x;

```

Comment:
if it's null... then why delete?

at this point this whole `else {...}` part is obsolete